src/Sarif/Schemata/sarif-schema.json

```diff
@@ -711,7 +723,8 @@
         "graphs": {
           "description": "An array of external property files containing a run.graphs object to be merged with the root log file.",
           "type": "array",
-          "minItems": 1,
+          "minItems": 0,
+          "default": [],
```

Comment:
this is wrong. it doesn't make sense to allow for a non-null and empty graphs collection. this is just empty data.

minItems:1 is right.